react.gradle

```diff
@@ -159,12 +159,21 @@ afterEvaluate {
             def devEnabled = !(config."devDisabledIn${targetName}"
                 || targetName.toLowerCase().contains("release"))
 
-            def extraArgs = config.extraPackagerArgs ?: [];
+            def extraArgs = []
```

Comment:
I changed the order of adding to extraArgs here so it matches the iOS script. This way it would still be possible to override args that are added here with `config.extraPackagerArgs`.